helpers.js

```diff
@@ -66,8 +66,11 @@ async function tryDelete(path) {
   try {
     const gitKey = await GitKey.fromRepoFolder(newpath);
     if (gitKey) { await gitKey.dispose(); }
-    fs.rmdirSync(newpath, { recursive: true });
-  } catch (err) { throw new Error(`Couldn't delete the repository: ${newpath}`); }
+  } catch (err) { } finally {
+    try {
+      fs.rmdirSync(newpath, { recursive: true });
+    } catch (err) { throw new Error(`Couldn't delete the repository: ${newpath}, ${err}`); }
+  }
```

Comment:
I don't understand this code. What is tried? which code can actually throw here - `GitKey.fromRepoFolder`, `gitKey.dispose` or `rmdirSync`? I know that all fs methods can throw and this should be tried, but I hate idea of try-catch block in finally. If catch should be empty, then let's make it empty and delete the finally – if nothing is thrown / returned in catch, then there's no need to even call finally.

Reply:
I was just trying to satisfy the linter with minimal impact. I didn't think to question the finally. But you are right, this was silly.